README.md

```diff
@@ -1,2 +1,13 @@
 # MADiE models
 Java based MADiE models used by MADiE microservices
+
+## Usage
+```
+<dependency>
+  <groupId>gov.cms.madie</groupId>
+  <artifactId>madie-java-models</artifactId>
+  <version>X.Y.Z</version>
+</dependency>
+```
+Where,\
+**X**: Measure version , **Y**: Minor version, **Z**: Patch version 
```

Comment:
`Measure` or `Major`?

Reply:
only measures and fhir(no major and fire anymore)

Reply:
corrected now, thanks!